crates/fraud_proofs/global_merkle_root/storage/src/update.rs

```diff
@@ -167,16 +170,37 @@ where
 
         self.store_processed_transaction(tx_id)?;
 
-        // TODO(#2583): Add the transaction to the `ProcessedTransactions` table.
         // TODO(#2585): Insert uplodade bytecodes.
         if let Transaction::Blob(tx) = tx {
             self.process_blob_transaction(tx)?;
         }
-        // TODO(#2587): Insert raw code for created contracts.
+        if let Transaction::Create(tx) = tx {
+            self.process_create_transaction(tx)?;
+        }
 
         Ok(())
     }
 
+    fn process_create_transaction(&mut self, tx: &Create) -> anyhow::Result<()> {
+        let bytecode_witness_index = tx.bytecode_witness_index();
+        let witnesses = tx.witnesses();
+        let bytecode = witnesses[usize::from(*bytecode_witness_index)].as_vec();
```

Comment:
If `bytecode_witness_index` is malicious, it will panic

Reply:
Interesting that clippy missed this :thinking: thanks for spotting!